src/promptflow-devkit/promptflow/_sdk/data/docker_csharp/README.md

```diff
@@ -7,5 +7,25 @@ Exported Dockerfile & its dependencies are located in the same folder. The struc
 - settings.json: a json file to store the settings of the docker image
 - README.md: the readme file to describe how to use the dockerfile
 
+
+Build Docker image:
+`docker build <build_output> -t <image_name>`
+
+
+**Run dag flow** with docker run by flask serving engine:
+`docker run -p 8080:8080 -e OPEN_AI_CONNECTION_API_KEY=<secret-value> -e PROMPTFLOW_WORKER_NUM=<expect-worker-num> -e PROMPTFLOW_WORKER_THREADS=<expect-thread-num-per-worker> <image_name>`
+example:
+`docker run -p 8080:8080 -e OPEN_AI_CONNECTION_API_KEY=111 -e PROMPTFLOW_WORKER_NUM=1 -e PROMPTFLOW_WORKER_THREADS=1 dag_flow_image`
+
+**Run flex flow** with docker run by flask serving engine:
```

Comment:
@elliotzh to take a look

Reply:
hmm I don't think this PR has covered csharp flows actually... chenyang will need to update the logic to get connection names for csharp flow to support that

Reply:
There is indeed no support for csharp, the next PR will support it.